DataStructures/BitArray/BitArray.cs

```diff
@@ -0,0 +1,1220 @@
+﻿/**
```

Comment:
Please change your editor config to match the project. The tab size should be equal to 4 spaces. Please change the tabs in this file to 4 spaces long.